solvers/+drakeFunction/+geometry/doc_cartesian2cylindrical.tex

```diff
@@ -0,0 +1,17 @@
+\documentclass{article}
```

Comment:
can we put these docs in the drake/doc directory instead?  (and update the reference to them in the comments).